config-provisioning/src/main/java/com/yahoo/config/provision/ClusterMembership.java

```diff
@@ -22,16 +22,28 @@ public class ClusterMembership {
     private ClusterMembership(String stringValue, Version vespaVersion, Optional<DockerImage> dockerImageRepo,
                               ZoneEndpoint zoneEndpoint) {
         String[] components = stringValue.split("/");
-        if (components.length < 4)
+        if (components.length < 3)
```

Comment:
```suggestion
        if (components.length < 4)
```

Reply:
I wanted to support everything this can output now ...

Reply:
So ideally, this should roll out first, since upgrades aren't atomic.

Reply:
Ah, well, no, the converse to this should roll out first, i.e, when an upgraded server writes a spec with `//`, the other servers should still be able to handle that meaningfully. They won't now, will they?

Reply:
Here.